theano/compile/mode.py

```diff
@@ -65,14 +65,24 @@ def register_linker(name, linker):
 OPT_FAST_COMPILE.name = 'OPT_FAST_COMPILE'
 OPT_STABILIZE.name = 'OPT_STABILIZE'
 
+OPT_O2 = gof.Query(include=['fast_compile', 'fast_compile_gpu', 'fusion'],
```

Comment:
Better to use OPT_FAST_COMPILE.including("fusion") so you don't have to modify both OPT_FAST_COMPILE and OPT_O2 if something is added to OPT_FAST_COMPILE. We wan't to keep O2 and FAST_COMPILE in sync.